Olha_Lesiuk/task_1.py

```diff
@@ -0,0 +1,15 @@
+list_of_elements = [1, 2, 3, 4, 5, 6, 7, 8]
+Oddlist = []
+EvenList = []
+
+for i in list_of_elements:
+    if i%2 == 0:
+        EvenList.append(i)
+    else:
+        Oddlist.append(i)
+
+EvenList = list(enumerate(EvenList))
+Oddlist = list(enumerate(Oddlist))
+
+print("Even numbers", EvenList)
+print("Odd numbers", Oddlist)
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/187702355-15879bf8-73b0-4279-af7c-b6984d1fe160.png)
This means that you do not have empty line at the end